src/sentry/eventstore/snuba/backend.py

```diff
@@ -48,20 +48,59 @@ class SnubaEventStorage(EventStorage):
     def get_events(
         self,
         filter,
-        additional_columns=None,
         orderby=None,
         limit=DEFAULT_LIMIT,
         offset=DEFAULT_OFFSET,
         referrer="eventstore.get_events",
     ):
         """
-        Get events from Snuba.
+        Get events from Snuba, with node data loaded.
         """
+        return self.get_events(
```

Comment:
is this a call to __get_events instead ?

Reply:
uhhh, yes it should be